config/main.py

```diff
@@ -1999,6 +2002,20 @@ def override_config_db(config_db, config_input):
     click.echo("Overriding completed. No service is restarted.")
 
 
+def yang_hard_dependency_check(config_json):
```

Comment:
>yang_hard_dependency_check [](http://example.com/codeflow?start=4&length=26)

There is no yang-model usage, so the function names are not accurate.

Reply:
Changed to table_hard_depdency_check